packages/wallets/src/components/WalletTourGuide/WalletTourGuide.tsx

```diff
@@ -20,12 +28,20 @@ const WalletTourGuide = () => {
     const [addMoreWalletsTransformValue, setAddMoreWalletsTransformValue] = useState('');
     const { isMobile } = useDevice();
 
+    // just because someone clicked button in dtrader and set local storage
+    // does not mean we should run the tour as we might need to wait for the account to be switched
+    const [run, setRun] = useState<boolean>(false);
+
     const switchWalletAccount = useWalletAccountSwitcher();
     const { isFetching, isLoading, isSuccess } = useAuthorize();
     const { data: wallets } = useWalletAccountsList();
     const { data: activeWallet } = useActiveWalletAccount();
     const { data: availableWallets } = useAllWalletAccounts();
 
+    const { isLoading: ctraderIsLoading } = useCtraderAccountsList();
+    const { isLoading: dxtradeIsLoading } = useDxtradeAccountsList();
+    const { isLoading: sorteAccountsListLoading } = useSortedMT5Accounts();
```

Comment:
a typo? 👀 
also, why not name this `mt5IsLoading` to not adhere from the prev 2 lines`? 

```suggestion
    const { isLoading: sortedAccountsListLoading } = useSortedMT5Accounts();
```

Reply:
fixed typo, updated name, ;)